benchmark/benchmark.js

```diff
@@ -1,11 +1,15 @@
 import { default as React } from "react";
-import { renderToString as renderToStringReact } from "react-dom/server";
+import { renderToString } from "react-dom/server";
 import { range } from "lodash";
 
-import { renderToString } from "../src";
+import { render } from "../src";
 import { time } from "./_util";
 
 
+// Make sure React is in production mode.
+process.env.NODE_ENV = "production";
```

Comment:
Yeah it seemed like it made the appropriate change, but I figure setting the ENV variable before anything executes would future-proof it in case there were any future changes to React (or any consumed module in `benchmark`) that implemented conditional exports based on `process.env.NODE_ENV`, which isn't unheard of.